dev-requirements.txt

```
@@ -9,6 +9,7 @@ cryptography
 html5lib  # needed for beautifulsoup
 jupyterlab >=3
 mock
+nbclassic
```

Does this make sense? If not maybe you can tweak the comment to make sense or so. If nbclassic isn't directly imported as a dependency in tests, I think a comment is very helpful to have to clarify the situation.

```suggestion
nbclassic  # ensures /tree/ handler is present as assumed by tests
```